project-workflow-trigger-options_5d42dcce042863478675bb99.md

```diff
@@ -31,13 +33,28 @@ Semaphore will create a workflow for every push to your repository.
 In every job from this workflow, Semaphore will export
 `SEMAPHORE_GIT_REF_TYPE` environment variable with value `branch`.
 
+## Whitelist branches
+
+By default, Semaphore will create a workflow for every branch. You can whitelist
+the branches you want to run, specifying their names, regular expressions
```

Comment:
You can whitelist the branches that you want to run by specifying their names separated by commas. Regular expressions enclosed with `/` are also allowed, for example: `regexexample`.